www/includes/easyparliament/templates/html/ni/index.php

```diff
@@ -0,0 +1,111 @@
+    <div class="hero">
+        <div class="row">
+        <div class="hero__mp-search">
+            <div class="hero__mp-search__wrap">
+                <h1>Do your MLAs represent you?</h1>
```

Comment:
Well, the Scottish one only shows you your non list MSP. I guess we could list all of them but I think I'd want some design time for that. I might open a ticket about it.